app/src/main/java/org/mozilla/firefox/vpn/device/ui/DevicesAdapter.kt

```diff
@@ -54,13 +52,7 @@ class DevicesAdapter(
             .from(parent.context)
             .inflate(R.layout.item_device, parent, false)
 
-        val holder = DevicesViewHolder(itemView)
-
-        holder.itemView.delete.setOnClickListener {
-            onDeleteClicked(holder)
-        }
-
-        return holder
+        return DevicesViewHolder(itemView) { holder -> onDeleteClicked(holder) }
```

Comment:
I think this can be written as
`return DevicesViewHolder(itemView, ::onDeleteClicked)`